site/gatsby-site/src/components/LayoutHideSidebar.js

```diff
@@ -79,7 +80,7 @@ const LayoutHideSidebar = ({ children, location, menuCollapseCallback, className
   return (
     <>
       <Header />
-      <Wrapper className={className}>
+      <Wrapper className={`grow ${className || ''}`}>
```

Comment:
nitpick: it looks like the `||` is not needed because, by default, `className` is empty already